inception-project/inception
# #1132 Unify training data selection for evaluation

During evaluation, different recommenders select different amounts of training data given the same data and the same DataSplitter. To make recommender evaluation more comparable, the used training data is now given as ratio compared to all training data of the selected recommender for the evaluation simulation.

**What's in the PR**
* Implemented ratio of used training data as field in the evaluation result.
* Also added a message field to the result object to communicate possible errors to the user via the learning curve chart.

**How to test manually**
* Configure a project with a recommender and annotated data.
* Start an evaluation simulation on the recommender tab in the project settings.
* Check that the learning curve on the annotation page is still properly shown.

**Unrelated Changes**
* Update to admin guide regarding the learning.curve flag.
